FtcRobotController/src/main/java/com/qualcomm/ftcrobotcontroller/opmodes/FtcOpModeRegister.java

```diff
@@ -56,11 +56,15 @@ public void register(OpModeManager manager) {
      */
 
     manager.register("NullOp", NullOp.class);
-    manager.register("K9TeleOp", K9TeleOp.class);
-    manager.register("K9TankDrive", K9TankDrive.class);
-    manager.register("K9Line", K9Line.class);
-    manager.register("K9IrSeeker", K9IrSeeker.class);
-    manager.register("K9AutoTime", K9AutoTime.class);
+    //manager.register("K9TeleOp", K9TeleOp.class);
```

Comment:
As a rule, I am not a fan of commented out code. I know it makes sense here. 

It would be nice if adding a function did not require us to add a line of code here. Not sure if there is a nice way to streamline that? Without streamlining it, I predict many merge conflicts within this file in the future.